packages/yoga/src/Avatar/Avatar.theme.js

```diff
@@ -0,0 +1,3 @@
+const Avatar = () => ({});
```

Comment:
Hi Le,
If you don't see the need to themify the avatar, you don't need this file
You can look at the 'chip' component, it doesn't have a theme file because it inherits all from base